build.cmd

```diff
@@ -501,6 +501,18 @@ if not defined __SkipCoreLibBuild (
 	    echo     %__BuildErr%
 	    exit /b 1
 	)
+
```

Comment:
We should switch this to just call a general packages.builds file so we don't have to explicitly list all the different builds files in the build.cmd/sh file.